holmes/core/supabase_dal.py

```
@@ -767,6 +771,80 @@ def get_issue_data(self, issue_id: Optional[str]) -> Optional[Dict]:
+        if isinstance(data, list):
+            data = data[0] if data else None
+        return data or {}
+
+    def get_holmes_chat_allowed_clusters(
+        self, user_id: Optional[str]
+    ) -> Optional[List[str]]:
+        """Return the clusters a user may access via Holmes chat (MA_HOLMES_CHAT).
+
+        The return value follows relay's RBAC contract:
+
+          * ``None``    – the user is a regular account user: full access, no
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)